spec/features/visitor/atom_feed_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
+feature 'account using a custom domain' do
```

Comment:
Only having a feature spec goes against the testing pyramid approach of the test suite. I know Rails 5 continues to blur the lines between these sorts of tests. But I think there is value in being able to run only the controller specs and already sort of know that *our* code works and having feature specs to ensure everything works together as a whole.

Controller specs are also a good place to focus a bit more on authorization concerns. While I would like the feature spec to verify the positive assumption that there is indeed a working feed feature, the controller level would be the place to test that the feeds really only contain the entries of the given account - something that is not checked by any spec right now and - in my opinion - should not be checked in a feature spec.